doc/reference/tooling/tt_cli/export.rst

```diff
@@ -11,12 +11,17 @@ Exporting data
 
 ..  code-block:: console
 
-    $ tt [crud] export URI SPACE:FILE ... [EXPORT_OPTION ...]
+    $ tt [crud|tdg2] export URI SPACE:FILE ... [EXPORT_OPTION ...]
 
-``tt [crud] export`` exports a space's data to a file.
-The ``crud`` command is optional and can be used to export a cluster's data by using the `CRUD <https://github.com/tarantool/crud>`_ module. Without ``crud``, data is exported using the :ref:`box.space <box_space>` API.
+``tt [crud|tdg2] export`` exports a space's data to a file. Three export commands
+cover the following cases:
 
-``tt [crud] export`` takes the following arguments:
+*   ``tt export`` exports data of a single-replicaset storage using the :ref:`box.space <box_space>` API.
+*   ``tt crud export`` exports data from a sharded cluster through its router using the `CRUD <https://github.com/tarantool/crud>`_ module.
+*   ``tt tdg2 export`` exports data from a `Tarantool Data Grid 2 <https://www.tarantool.io/ru/tdg/latest/>`_ cluster
+    through its router.
```

Comment:
TDG2 router roles are called "Connector".

You can also say here that `export/import` is using the [repository API](https://www.tarantool.io/en/tdg/latest/reference/sandbox/repository-api/#repository-api).